pipelines/jobs/Semeru_configuration/jdk11u_pipeline_config.groovy

```diff
@@ -356,7 +356,13 @@ class Config11 {
                         'extended.jck.fips140_3_OpenJCEPlusFIPS',
                         'special.jck.fips140_3_OpenJCEPlusFIPS',
                         'sanity.openjdk.fips140_3_OpenJCEPlusFIPS',
-                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS'
+                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS',
+                        'extended.functional.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
+                        'sanity.jck.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
+                        'extended.jck.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
+                        'special.jck.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
+                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
+                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS.FIPS140-3'
```

Comment:
To confirm, this is only enabling on xlinux ce jdk11. Can we update the commit comment to include that detail?

Reply:
For now, yes, just xlinux ce jdk11. We will need to enable it for JDK11+ LTS CE. See [internal issue](https://github.ibm.com/runtimes/automation/issues/106).

I would like to ensure this PR works on xlinux CE jdk11 first before we enable it on all supported platforms and versions.